consensus/hotstuff/safetyrules/safety_rules_test.go

```diff
@@ -286,31 +286,20 @@ func (s *SafetyRulesTestSuite) TestProduceVote_PersistStateException() {
 
 // TestProduceVote_VotingOnUnsafeProposal tests different scenarios where we try to vote on unsafe blocks
 func (s *SafetyRulesTestSuite) TestProduceVote_VotingOnUnsafeProposal() {
-	s.Run("invalid-block-view", func() {
-		// create block with block.View == block.QC.View
+
+	s.Run("happy-path-includes-last-view-tc", func() {
```

Comment:
```suggestion
	// a proposal which includes a QC for the previous round should not contain a TC
	s.Run("proposal-includes-last-view-qc-and-tc", func() {
```
It wasn't clear to me why this was named "happy path"